internal/services/recoveryservices/site_recovery_replicated_vm_resource.go

```diff
@@ -318,61 +321,162 @@ func networkInterfaceResource() *pluginsdk.Resource {
 				ValidateFunc: azure.ValidateResourceID,
 			},
 
-			"failover_test_static_ip": {
-				Type:         pluginsdk.TypeString,
-				Optional:     true,
-				Computed:     true,
-				ForceNew:     false,
-				ValidateFunc: validation.StringIsNotEmpty,
-			},
+			"ip_configuration": {
+				Type:       pluginsdk.TypeList,
+				Optional:   true,
+				ConfigMode: pluginsdk.SchemaConfigModeAttr,
```

Comment:
We have to specify `configMode` for it, otherwise there will be a runtime error:
```
resource azurerm_site_recovery_replicated_vm: ip_configuration: in *schema.Resource with ConfigMode of attribute, so must also have ConfigMode of attribute
```